applications/dir/prod/values.yaml

```diff
@@ -0,0 +1,274 @@
+      # Since we are the bootstrap node, this can be left empty.
+      # bootstrap_peers:
+      #   - /ip4/1.1.1.1/tcp/1
+      #   - /ip4/1.1.1.1/tcp/2
+
+      # GossipSub configuration for efficient label announcements
+      # When enabled, labels are propagated via GossipSub mesh to ALL subscribed peers
+      # When disabled, falls back to DHT+Pull mechanism (higher bandwidth, limited reach)
+      # Default: true (recommended for production)
+      gossipsub:
+        enabled: true
+
+    # Sync configuration
+    sync:
+      # How frequently the scheduler checks for pending syncs
+      scheduler_interval: "30s"
+      
+      # Maximum number of sync workers running concurrently
+      worker_count: 1
+      
+      # Timeout for individual sync operations
+      worker_timeout: "10m"
+      
+      # Registry monitor configuration
+      registry_monitor:
+        check_interval: "30s"
+      
+      # Authentication configuration for sync operations
+      auth_config:
+        username: "user"
+        password: "user"
+
+    # Publication configuration
+    publication:
+      # How frequently the scheduler checks for pending publications
+      scheduler_interval: "1h"
+      
+      # Maximum number of publication workers running concurrently
+      worker_count: 1
+      
+      # Timeout for individual publication operations
+      worker_timeout: "30m"
+
+  # Extra environment variables for the apiserver container.
+  # Add ZOT CA certs here for OCI registry TLS verification.
+  extraEnv:
+  - name: SSL_CERT_DIR
+    value: "/etc/ca-certs"
+
+  # Extra volume mounts for the apiserver container
+  extraVolumeMounts: 
+  - name: zot-config-storage
+    mountPath: /etc/zot
+  - name: dir-routing-storage
+    mountPath: /etc/routing
+  - name: dir-ca-certs
+    mountPath: /etc/ca-certs
+
+  # Extra volumes
+  extraVolumes: 
+  # Zot config storage volume for config hot-reloading
+  - name: zot-config-storage
+    hostPath:
+      path: /opt/zot-config
+      type: DirectoryOrCreate
+  # Routing storage volume for peer-to-peer data
+  - name: dir-routing-storage
+    emptyDir: {}
+  # CA certs for Zot registry TLS verification
+  - name: dir-ca-certs
+    secret:
+      secretName: prod-zot-directory-outshift-test-tls
+      items:
+      - key: ca.crt
+        path: zot-ca.crt
+
+  # Zot registry configuration (subchart)
+  zot:
+    # Expose the Zot registry via Ingress for external access (with TLS termination)
+    ingress:
+      enabled: true
+      className: "nginx"
+      annotations:
+        cert-manager.io/cluster-issuer: letsencrypt
+      hosts:
+        - host: prod.zot.directory.outshift.test
+          paths:
+            - path: /
+              pathType: ImplementationSpecific
+      tls:
+        - secretName: prod-zot-directory-outshift-test-tls # legit:ignore
+          hosts:
+            - prod.zot.directory.outshift.test
+
+    extraVolumeMounts: []
+    extraVolumes: []
```

Comment:
Remove, because we're using persistent storage for Zot. ZOT Chart v0.1.87 has built-in persistence support.
```suggestion
```